amd_openvx_extensions/amd_media/decoder.cpp

```diff
@@ -873,6 +891,9 @@ void CLoomIoMediaDecoder::DecodeLoop(int mediaIndex)
             decodeFrameCount[mediaIndex]++;
             PushAck(mediaIndex, 0);
         }
+#if ENABLE_PERF_MEASURE
```

Comment:
remove this code line (894-896) and put after this [line](https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/blob/master/amd_openvx_extensions/amd_media/decoder.cpp#L725) to print the final average time